src/crud.js

```diff
@@ -1,31 +1,65 @@
-import './style.css';
+const listTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const listCard = document.getElementById('list');
+const insertInput = document.getElementById('insert');
+const enterBtn = document.getElementById('enterBtn');
 
-const listTasks = [
-  {
-    description: 'Going to Office',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Meeting with ali',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Go to market',
+const updateIndexes = () => {
+  listTasks.forEach((task, index) => {
+    task.index = index;
+  });
+};
+
+const updateLocalStorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(listTasks));
+};
+
+const deleteTask = (index) => {
+  listTasks.splice(index, 1);
+  updateIndexes();
+  updateLocalStorage();
+};
```

Comment:
- [ ] When the delete button is clicked and the todo is deleted the screen is not updated to reflect the deleted todo. The deleted todo is only removed from the local storage but not the screen. I kindly suggest you fix this bug as it is not supposed to function this way. At every click of the delete button the todo should be removed from the screen and the local storage updated as well